### PR TITLE
[SYCL] enable a subdevice test for numa affinity.

### PR DESCRIPTION
**Pull request by glyons-intel**
Signed-off-by: Gail Lyons <gail.lyons@intel.com>

### CODE REVIEW

File: SYCL/Basic/query.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env CreateMultipleSubDevices=2 env EnableTimestampPacket=1 \
```
Comment (smaslov-intel) [inline SYCL/Basic/query.cpp:2]: I thought the CreateMultipleSubDevices=2 would not work on DG1/ATS with 1 tile
What kind of GPU HW do we tests with ?

**Review by smaslov-intel — DISMISSED** [review]
LGTM

File: SYCL/Basic/query.cpp (hunk 1 above)
Comment (vladimirlaz) [reply SYCL/Basic/query.cpp:2]: @smaslov-intel, @glyons-intel  We are using GEN9 machines. But the test can be run on DG1/ATS as well. Please mark as UNSUPPORTED: gpu-intel-dg1 to make sure that the test will not fail there (if it is exp[ected behavior) or XFAIL: gpu-intel-dg1 (if it is going to be fixed).

**Review by vladimirlaz — CHANGES_REQUESTED** [review]
LIT rules for DGI1 should be set

File: SYCL/Basic/query.cpp (hunk 1 above)
Comment (glyons-intel) [reply SYCL/Basic/query.cpp:2]: Done.

**Comment by glyons-intel** [comment]
> LIT rules for DGI1 should be set

Done.

**Review by smaslov-intel — APPROVED** [review]

**Review by vladimirlaz — APPROVED** [review]

**Pull request merged**